### PR TITLE
python311Packages.msldap: 0.5.6 -> 0.5.7, python311Packages.aiosmb: 0.4.7 -> 0.4.8

### PR DESCRIPTION
**Pull request by fabaff**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 259330` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>18 packages built:</summary>
  <ul>
    <li>crackmapexec</li>
    <li>crackmapexec.dist</li>
    <li>python310Packages.aiosmb</li>
    <li>python310Packages.aiosmb.dist</li>
    <li>python310Packages.lsassy</li>
    <li>python310Packages.lsassy.dist</li>
    <li>python310Packages.msldap</li>
    <li>python310Packages.msldap.dist</li>
    <li>python310Packages.pypykatz</li>
    <li>python310Packages.pypykatz.dist</li>
    <li>python311Packages.aiosmb</li>
    <li>python311Packages.aiosmb.dist</li>
    <li>python311Packages.lsassy</li>
    <li>python311Packages.lsassy.dist</li>
    <li>python311Packages.msldap</li>
    <li>python311Packages.msldap.dist</li>
    <li>python311Packages.pypykatz</li>
    <li>python311Packages.pypykatz.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/aiosmb/default.nix [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-ze8x0vFGnPAIQQicuJxAcBVEeeKOGUHvepRTO4Ejx+g=";
+    hash = "sha256-siV2cr8YuU4kWRNHGlV29RjeHxCk6mcbK7J15LZguls=";
   };
 
   propagatedBuildInputs = [
```
Comment (natsukium) [inline pkgs/development/python-modules/aiosmb/default.nix:29]: I guess we need to add cryptography to propagatedBuildInputs.
https://github.com/skelsec/aiosmb/blob/7d50c87bd62920284f0b712aa3e5b1df9539e752/setup.py#L52

Comment (fabaff) [reply pkgs/development/python-modules/aiosmb/default.nix:30]: Added, thanks for checking.

**Review by natsukium — APPROVED** [review]
LGTM, thanks.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 259330` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>18 packages built:</summary>
  <ul>
    <li>crackmapexec</li>
    <li>crackmapexec.dist</li>
    <li>python310Packages.aiosmb</li>
    <li>python310Packages.aiosmb.dist</li>
    <li>python310Packages.lsassy</li>
    <li>python310Packages.lsassy.dist</li>
    <li>python310Packages.msldap</li>
    <li>python310Packages.msldap.dist</li>
    <li>python310Packages.pypykatz</li>
    <li>python310Packages.pypykatz.dist</li>
    <li>python311Packages.aiosmb</li>
    <li>python311Packages.aiosmb.dist</li>
    <li>python311Packages.lsassy</li>
    <li>python311Packages.lsassy.dist</li>
    <li>python311Packages.msldap</li>
    <li>python311Packages.msldap.dist</li>
    <li>python311Packages.pypykatz</li>
    <li>python311Packages.pypykatz.dist</li>
  </ul>
</details>

**Pull request merged**